frontend/src/store/FamLoginUserState.ts

```diff
@@ -207,7 +207,24 @@ const getCachedAppRolesForDelegatedAdmin = (
             ? -1
             : 1;
     });
-}
+};
+
+const getMyCachedPermissions = () => {
```

Comment:
Hi Jota, I think this method is still not correct. Not sure if you tested locally, but when I have just one FAM_ADMIN role, it's not showing in this table. 
<img width="1716" alt="Screen Shot 2024-04-05 at 10 07 26 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/c53d7572-07e1-4cdf-82ec-f14cf755ff3e">

And when I have FAM_ADMIN and FOM_ADMIN access, this is also not correct. 
<img width="1625" alt="Screen Shot 2024-04-05 at 10 08 09 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/72b33b75-7187-4185-8165-6ca9938ce850">

So the `access` example you can check our confluence page at the bottom section https://apps.nrs.gov.bc.ca/int/confluence/display/FSAST1/Delegated+Access+Administration+Design, basically it will have three types, you need to first check the `auth_key`, auth_key can be "FAM_ADMIN", "APP_ADMIN" and "DELEGATED_ADMIN", and we need to handle them separately

First of all, you want to show an example of you data structure, like what you'll pass to the table component. And then for each auth key cases, we need to grab the information and format them 
1. If auth_key is FAM_ADMIN, then you just find the FAM's application description from the grants, filter the grant and return the application where name is FAM, the role name will just be `ADMIN` (hardcode this)
2. If auth_key is APP_ADMIN, then you want the information of each application in the grants, the role name will be `ADMIN` as well, no need to check the
3. If auth_key is DELEGATED_ADMIN, then for each grant, you want to application information as well as the role information

Reply:
oh, I see. I'll fix it following your comment